lisa/commands.py

```diff
@@ -54,12 +55,14 @@ def run(args: Namespace) -> int:
 
 # check runbook
 def check(args: Namespace) -> int:
-    load_runbook(args.runbook, args.variables)
+    builder = RunbookBuilder.build_from_path(args.runbook, args.variables)
```

Comment:
Here and below, why not just call build_from_path() and ignore the return?

Reply:
This is to create a builder from the class method, not an instance method.